tests/pytest_tests/system_tests/test_artifacts/test_wandb_artifacts_full.py

```diff
@@ -236,6 +236,39 @@ def dir_size():
     assert dir_size() == 0
 
 
+def test_mutable_uploads(tmp_path, monkeypatch):
```

Comment:
These tests will be updated in the follow up PR that makes caching optional https://github.com/wandb/wandb/pull/7098 to include combinations of policy selections with cache selections.